.circleci/config.yml

```diff
@@ -176,14 +189,26 @@ workflows:
   version: 2
   build_and_test:
     jobs:
-      # Set up environment variables
-      - configure_env_vars
-      # Install dev dependencies and do simple tests (sniff, unit tests, etc.)
-      - unit
-      # Build deploy and test on target platform
-      - functional
-      # Deploy before running visual test
-      - visual_regression_test:
+        # Set up environment variables
+        - configure_env_vars
+        # Install dev dependencies and do simple tests (sniff, unit tests, etc.)
+        - unit
+        # Build deploy and test on target platform
+        - functional
+        # Deploy before running visual test
+        - visual_regression_test:
             requires:
                 - configure_env_vars
                 - functional
+        # Composer lock updater
+        - composer_lock_updater
+#  auto_update_dependencies:
+#    triggers:
+#        - schedule:
+#            cron: "5 4 * * *"
```

Comment:
This is five minutes after 4am UTC every day? Since Circle does not support anything beyond the most basic cron schedule syntax, our choice is to run daily or monthly.

Reply:
> This is five minutes after 4am UTC every day?

Yes.

> Since Circle does not support anything beyond the most basic cron schedule syntax, our choice is to run daily or monthly.

That is not true. This is pulled from [`pantheon-systems/example-terminus-auto-update-script`](https://github.com/pantheon-systems/example-terminus-auto-update-script) where it does, indeed, run at `4:05` daily.

See [this CircleCI docs](https://circleci.com/docs/2.0/workflows/#specifying-a-valid-schedule). Values for all 5 places are supported but Cron step syntax is not.

Anyway, I updated it to hourly. Once this PR passes I will remove the job from the workflow that runs on commit and uncomment the cron workflow.

Reply:
You're right, hourly is also an option, but that's too frequent. I thought daily was too frequent, and was hoping for weekly, but alas, that is not supported. Given the option between every minute, hourly, daily and monthly (or yearly), I think we should do daily.

We could have our script no-op one time out of seven, but I don't want to add the complexity.